models/src_models_zendesk.yml

```diff
@@ -131,15 +134,16 @@ models:
         description: Boolean field indicating if the comment is a twitter tweet
       - name: is_voice_comment
         description: Boolean field indicating if the comment is a voice comment
-      - name: from
+      - name: from_number
         meta:
           sensitive: true
-      - name: to
+      - name: to_number
         meta:
```

Comment:
`to_number` is a snowflake [function](https://docs.snowflake.com/en/sql-reference/functions/to_decimal.html)

Reply:
ok, makes sense

Reply:
fixed